integration-tests/gradle/src/test/java/io/quarkus/gradle/FastJarFormatWorksTest.java

```diff
@@ -20,6 +31,51 @@ public void testFastJarFormatWorks() throws Exception {
         assertThat(quarkusApp).exists();
         Path jar = quarkusApp.resolve("quarkus-run.jar");
         assertThat(jar).exists();
+
+        File output = new File(projectDir, "build/output.log");
+        output.createNewFile();
+        Process process = doLaunch(jar, output);
+        try {
+            // Wait until server up
+            dumpFileContentOnFailure(() -> {
+                await()
+                        .pollDelay(1, TimeUnit.SECONDS)
+                        .atMost(1, TimeUnit.MINUTES).until(() -> DevModeTestUtils.getHttpResponse("/hello", 200));
+                return null;
+            }, output, ConditionTimeoutException.class);
+
+            String logs = FileUtils.readFileToString(output, "UTF-8");
+
+            assertThatOutputWorksCorrectly(logs);
+            // test that the application name and version are properly set
+            getHttpResponse("/hello");
```

Comment:
You should probably assert that the response is `"hello"`

Reply:
Will do